apollo-federation/tests/query_plan/build_query_plan_tests/handles_fragments_with_directive_conditions.rs

```diff
@@ -72,7 +72,7 @@ fn fragment_with_intersecting_parent_type_and_directive_condition() {
 }
 
 #[test]
-fn nested_fragment_with_interseting_parent_type_and_directive_condition() {
+fn nested_fragment_with_intersecting_parent_type_and_directive_condition() {
```

Comment:
Do you need to create a new supergraph schema and delete the old one?

Reply:
Yeah. Forgot to regenerate it.